src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -98,6 +100,9 @@
     <Compile Include="$(CommonPath)\System\Net\SecurityStatusPal.cs">
       <Link>Common\System\Net\SecurityStatusPal.cs</Link>
     </Compile>
+    <Compile Include="..\ref\TlsCipherSuite.cs">
```

Comment:
Do we do this anywhere else in corefx?

There's value in having the ref and src be separate.  I do not think we should be including the ref into the src build like this.  We should keep them separate, and be very explicit about choosing to update the ref.

Reply:
move generation to src, copied cs to ref